src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int mrn = 100;
```

Comment:
What about constant naming? https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names 
And mrn - not informative constant name